src/test/java/de/undercouch/actson/JsonParserTest.java

```diff
@@ -189,6 +180,28 @@ public void emptyObject() {
     assertJsonObjectEquals(json, parse(json));
   }
 
+  @Test
```

Comment:
Please add comments to your new test methods. All other test methods have comments.